include/ola/client/ClientTypes.h

```diff
@@ -42,10 +42,14 @@ namespace client {
  */
 class OlaPlugin {
  public:
-  OlaPlugin(unsigned int id, const std::string &name, bool active)
+  OlaPlugin(unsigned int id,
+            const std::string &name,
+            bool active,
+            bool enabled = true)
```

Comment:
I think you can avoid the default here, since the only thing that should be creating plugin objects is the client itself.